### PR TITLE
add pysftp recipe

### PR DESCRIPTION
**Pull request by shaunwbell**

### CODE REVIEW

File: recipes/pysftp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - pysftp
+
+about:
+  home: https://bitbucket.org/dundeemt/pysftp
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'A friendly face on SFTP'
+
```
Comment (mwcraig) [inline recipes/pysftp/meta.yaml]: minor, but can you please remove the blank line?

**Comment by shaunwbell** [comment]
Hah! sorry, didn't mean to remove the example, that's no bueno... fixed. as is the space

File: recipes/pysftp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 2
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - paramiko >=1.17.0
```
Comment (synapticarbors) [inline recipes/pysftp/meta.yaml]: I think you can drop `paramiko` as a `build` requirement.

File: recipes/pysftp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 2
```
Comment (synapticarbors) [inline recipes/pysftp/meta.yaml]: Please keep the build `number` at `0`. You can increment it in the feedstock if you make a change that doesn't involve a change in version number of the package. No need to increment here.

**Comment by shaunwbell** [comment]
thanks for the reviews.  changes made.

**Pull request merged**